Portfolio.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE = html>
+<html lang="en">
+<head>
+  	<meta charset="utf-8">
+  	<title>Jenny Allton-Twist</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet">
+	<link href="Portfoliostyle.css" type="text/css" rel="stylesheet">	
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+	<nav>
+		<ul>
+			<li><a href="#home">Home</a></li>
+			<li><a href="#portfolio">Portfolio</a></li>
+			<li><a href="#aboutme">About Me</a></li>
+			<li><a href="#contact">Contact</a></li>
+		 </ul>
+		 <div>
+			 
+	 <!-- <div><img src="" alt=""/> Place for logo -->
+		 </div>
+	</nav>
+
+
+
+	<div class="intropage" id="home">
+		<div class="container">
+			<h1>JENNY ALLTON-TWI<span clasmate s="spinner">S</span>T</h1>
+		</div>
+	</div>
+
+
+
+	<div class="portfolio" id="portfolio"><h2>Portfolio</h2>
+		<div class="container">
+			<div class="box">
+				<a href="project.html"><img src="tech.jpg" alt="Project Image"/></a>
+				<p>Some info about this very cool thing I invented here</p>
```

Comment:
the 0 and the h4 are the wrong way round. the project title is a heading, so a `h4`, the date is just additional info, so a `p`

Reply:
I like the date and location as a bolded or more important thing, so I'm leaving as an h4 but I have put a heading h2 above the p now.